src/main/java/org/example/P03_RectanglePerimeterAndArea.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+public class P03_RectanglePerimeterAndArea {
+    public static void main(String[] args) {
+           /* Write a C program to compute the perimeter and area of a rectangle with a
```

Comment:
Move documentation outside class body. You can put the problem description at line 2.